core/src/test/java/org/apache/calcite/rel/logical/ToLogicalConverterTest.java

```diff
@@ -468,4 +468,19 @@ private void verify(RelNode rel, String expectedPhysical, String expectedLogical
             + "    LogicalTableScan(table=[[foodmart, employee]])\n";
     verify(rel(sql), expectedPhysial, expectedLogical);
   }
+
+  @Test public void testTableModify() {
+    String sql = "insert into \"employee\" select * from \"employee\"";
+    RelNode rel = rel(sql);
+    String expectedPhysial = ""
+        + "JdbcToEnumerableConverter\n"
+        + "  JdbcTableModify(table=[[foodmart, employee]], operation=[INSERT], flattened=[true])\n"
+        + "    JdbcTableScan(table=[[foodmart, employee]])\n";
+    String expectedLogical = ""
+        + "LogicalTableModify(table=[[foodmart, employee]], "
+        + "operation=[INSERT], flattened=[true])\n"
+        + "  LogicalTableScan(table=[[foodmart, employee]])\n";
+    verify(rel, expectedPhysial, expectedLogical);
+  }
```

Comment:
Inline the `rel(sql)`.

Reply:
> Inline the `rel(sql)`.

Thanks for you review, i have updated it.